File: Day Four/Python/credit_card_validator_main_app.py
from credit_card import *

card_number = input("Hello. Kindly Enter Card number to verify: ")

card_valdity_status = credit_card_validator(card_number)
card_length = credit_card_length(card_number)
card_type = credit_card_type(card_number)

print(f""" 
    Credit Card Type: {card_type}
    Credit Card Number: {card_number}
    Credit Card Digit Length: {card_length}
    Credit Card Validity Status: {card_valdity_status}
    
    """)

File: Day Four/Java/creditCardValidatorMainApp.java
import java.util.Scanner;
public class creditCardValidatorMainApp{

    public static void main(String[] args){
    
        creditCardValidatorFunction cardValidation = new creditCardValidatorFunction();
    
        Scanner input = new Scanner(System.in);
        System.out.print("Hello. Kindly Enter Card number to verify: ");
        String cardNumber = input.nextLine();

        String cardValdityStatus = cardValidation.creditCardValidator(cardNumber);
        int cardLength = cardValidation.creditCardLength(cardNumber);
        String cardType = cardValidation.creditCardType(cardNumber);

        System.out.print(String.format(""" 
            Credit Card Type: %s
            Credit Card Number: %s
            Credit Card Digit Length: %d
            Credit Card Validity Status: %s
            
            """, cardType, cardNumber, cardLength, cardValdityStatus
            ));
            
    }
    
}
